web/src/tools/testing/s_keyman_com.sh

```diff
@@ -1,12 +1,7 @@
 #!/usr/bin/env bash
-
 #
-# Prevents 'clear' on exit of mingw64 bash shell
+# TODO: is this script still necessary or is /web/ci.sh a complete replacement?
```

Comment:
This script was in its prior state due to the half-and-half state for the s.keyman.com build action.  At the time, we didn't want the `.sh` script to actually _commit_ the PR and changes... but prepping them was fine.

So, now that `./ci.sh prepare:s-keyman-com` is no longer just _preparing_ the files for the PR but also pushing the commit (and making the PR) itself... nah, should be fine to fully fold in the remaining bits into the `ci.sh` script.  It _was not_ a full replacement, but it seems your solution to #9886 is to _make_ (or... "has made"?) `ci.sh` a full replacement.

----

Re: this question and the one from #9886 -

> why is s_keyman_com.sh in web/src/tools/testing? Seems like the wrong place for this.

Ok, yeah, not sure why I ended up putting it there.  Does seem like more of a 'building' one, but... oh well.

Reply:
I briefly looked on TC Web configs. Apparently the Release (push release to s.keyman.com) is still using Powershell. :confused:

Reply:
s_keyman_com.sh seems redundant now -- it is never called afaict. Should we remove it?

Reply:
I've updated the TC Web configs to skip the git commit phase if .build-builder is present. We'll need to port this PR back to 16.0 as well, and then we can convert the entire script into a bash call.